app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt

```diff
@@ -116,22 +118,30 @@ class ProfileFragment : Fragment(), ProfileView, ActionMode.Callback {
             val personDrawable = DrawableHelper.getDrawableFromId(R.drawable.ic_person_black_24dp, this)
             val atDrawable = DrawableHelper.getDrawableFromId(R.drawable.ic_at_black_24dp, this)
             val emailDrawable = DrawableHelper.getDrawableFromId(R.drawable.ic_email_black_24dp, this)
+            val linkDrawable = DrawableHelper.getDrawableFromId(R.drawable.ic_link_black_24dp, this)
 
-            val drawables = arrayOf(personDrawable, atDrawable, emailDrawable)
+            val drawables = arrayOf(personDrawable, atDrawable, emailDrawable, linkDrawable)
             DrawableHelper.wrapDrawables(drawables)
             DrawableHelper.tintDrawables(drawables, this, R.color.colorDrawableTintGrey)
-            DrawableHelper.compoundDrawables(arrayOf(text_name, text_username, text_email), drawables)
+            DrawableHelper.compoundDrawables(arrayOf(text_name, text_username, text_email, text_avatar_url), drawables)
         }
     }
 
     private fun listenToChanges() {
-        Observables.combineLatest(text_name.asObservable(), text_username.asObservable(), text_email.asObservable()).subscribe({ t ->
-                    if (t.first.toString() != currentName || t.second.toString() != currentUsername || t.third.toString() != currentEmail) {
-                        startActionMode()
-                    } else {
-                        finishActionMode()
-                    }
-                })
+        Observables.combineLatest(text_name.asObservable(),text_username.asObservable()).subscribe({ t ->
```

Comment:
As I told you, take a look of how to observe more than three observables here.

Reply:
Ok @filipedelimabrito  I'll look into it and making changes accordingly..